scanner/models.py

```diff
@@ -199,7 +199,7 @@ def delete(self, *args, **kwargs):
         return deleted
 
     @property
-    def status(self):
+    def status(self) -> str:
```

Comment:
Not sure we want to mix and match type annotations. We should probably remove this.